base/ca/src/org/dogtagpki/ct/LogServer.java

```diff
@@ -0,0 +1,51 @@
+package org.dogtagpki.ct;
+
+import java.net.URL;
+
+public class LogServer {
+
+    private int id;
+    private String publicKey;
+    private int version;
+    private boolean enabled;
+    private URL url;
+
+    public URL getUrl() {
+        return url;
+    }
+
+    public void setUrl(URL url) {
+        this.url = url;
+    }
+
+    public int getId() {
+        return id;
+    }
+
+    public void setId(int id) {
+        this.id = id;
+    }
+    public String getPublicKey() {
+        return publicKey;
+    }
+
+    public void setPublicKey(String publicKey) {
+        this.publicKey = publicKey;
+    }
+
+    public int getVersion() {
+        return version;
+    }
+
+    public void setVersion(int version) {
+        this.version = version;
```

Comment:
since we do not support version 2 yet, could we disallow anything other than 1 here?

Reply:
Done